### DISCUSSION TITLE
Add tilt_angle to frameit

### DISCUSSION THREAD

**Discussion by bennoland** [fastlane/fastlane#16998] [Ideas]
I used frameit to generate my screenshots for the App Store (see attached). I wanted to add a little flair with a tilt on the first screenshot, but frameit doesn't currently support this so I had to do that one by hand.

![all-screens](https://user-images.githubusercontent.com/4206203/89450384-ad4dd400-d717-11ea-8449-51ddb4aff8cf.png)

**Comment by nysander** [comment]
I would like having option to generate something like that below image where tilt is in 2 axis

![billboard-kopia](https://user-images.githubusercontent.com/753327/89453247-3e946900-d75f-11ea-8301-9e65528f8895.png)

live in appstore looks like this: [bit.ly/codeconfapp](bit.ly/codeconfapp)

PS. if direct appstore url is not welcome please let me know I will remove it.
